config/symbols.us.stno0.txt

```diff
@@ -79,6 +79,7 @@ init_velocity_y = 0x801825C0;
 bone_projectile_velocity_x = 0x8018272C;
 sensors_special = 0x8018275C;
 sprites_nz0_3 = 0x801BEBB0;
+ctulhu_shockwave_uvs = 0x801C11B0;
```

Comment:
This sits in a later data section `0x27E0` so I've not imported this for now, just mapped the symbol.